.travis.yml

```diff
@@ -12,5 +12,16 @@ php:
   - 7.1
   - hhvm
 
+matrix:
+  include:
+    - php: 7
+      env: PHPSTAN=1
+    - php: 7.1
+      env: PHPSTAN=1
+
 before_script:
     - composer install -n
+
+script:
+  - vendor/bin/phpunit
+  - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.5.2 && vendor/bin/phpstan analyse -c phpstan.neon -l 3 lib tests; fi
```

Comment:
`phpstan/phpstan` installation should be part of `before_script` (also there: to be done conditionally)